.bitmap

```diff
@@ -11,267 +11,297 @@
 {
     "accordion": {
         "scope": "showtime.universal",
-        "version": "0.0.26",
+        "version": "0.0.33",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/accordion"
     },
     "activity-indicator": {
         "scope": "showtime.universal",
-        "version": "0.0.20",
+        "version": "0.0.27",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/activity-indicator"
     },
     "alert": {
         "scope": "showtime.universal",
-        "version": "0.0.38",
+        "version": "0.0.45",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/alert"
     },
     "avatar": {
         "scope": "showtime.universal",
-        "version": "0.0.23",
+        "version": "0.0.30",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/avatar"
     },
     "bottom-sheet": {
         "scope": "showtime.universal",
-        "version": "0.0.32",
+        "version": "0.0.39",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/bottom-sheet"
     },
     "button": {
         "scope": "showtime.universal",
-        "version": "0.0.30",
+        "version": "0.0.37",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/button"
     },
     "checkbox": {
         "scope": "showtime.universal",
-        "version": "0.0.27",
+        "version": "0.0.34",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/checkbox"
     },
+    "clamp-text": {
+        "scope": "showtime.universal",
+        "version": "0.0.4",
+        "mainFile": "index.ts",
+        "rootDir": "packages/design-system/clamp-text"
+    },
+    "collapsible-tab-view": {
+        "scope": "showtime.universal",
+        "version": "0.0.22",
```

Comment:
Just a type of issue that causes it to not have a dist folder, I've been digging for a long time. 
I didn't find the issue until version `v0.0.20`. 🤣